node/test/util/rebase_util.js

```diff
@@ -531,3 +529,90 @@ x=E:C3M-4 s=Sa:a1s,t=Sb:b1t;E;Bmaster=3M;
         });
     });
 });
+
+describe("listRebaseCommits", function () {
```

Comment:
Did you mean for this to be in this "deindent test cases" commit?

Reply:
whoops! Not sure how I managed that.

Reply:
I'm not sure what's going on here. These test cases were definitely in the parent of this commit (db6059c2daf650dd55efdfd79c5b50b631059961), but I do see the same thing with `git diff`. Any ideas, @novalis ?